Uplift.DataAccess/Data/Repository/Repository.cs

```diff
@@ -54,7 +54,7 @@ public IEnumerable<T> GetAll(Expression<Func<T, bool>> filter = null, Func<IQuer
             return query.ToList();
         }
 
-        public T GetFirstOrDefault(Expression<Func<T, bool>> filter = null, Func<IQueryable<T>, IOrderedQueryable<T>> orderBy = null, string includeProperties = null)
+        public T GetFirstOrDefault(Expression<Func<T, bool>> filter, string includeProperties = null)
```

Comment:
`public T GetFirstOrDefault(Expression<Func<T, bool>> filter = null, string includeProperties = null)`

Not sure just I think the filter should be equal to null, By this way the user doesn't have to define it. He can but it is not mandatory.

Reply:
I saw this Error:
![image](https://github.com/user-attachments/assets/3aa63f9d-1b97-4887-8baa-89df39b5dcf3)